resource-managers/kubernetes/integration-tests/src/test/resources/log-config-test-log4j.properties

```diff
@@ -23,4 +23,4 @@ appender.console.type = Console
 appender.console.name = console
 appender.console.target = SYSTEM_ERR
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %d{yy/MM/dd HH:mm:ss} %p %c: %m%n%ex
+appender.console.layout.pattern = %d{HH:mm:ss.SSS} %p %c: %maxLen{%m}{512}%n%ex{8}%n
```

Comment:
BTW, do you know why this property file name is not `log4j2.properties`?

Reply:
It's loaded manually in SparkConfPropagateSuite